Helper/Config.php

```diff
@@ -735,4 +735,20 @@ public function getToggleCheckout()
     {
         return $this->getAdditionalConfigProperty('toggleCheckout');
     }
+
+    /**
+     * Get Bolt additional configuration for Amasty Gift Card support, stored in the following format:
+     *
+     * {
+     *   "amastyGiftCard": {
+           "payForEverything": true|false   // true, the default,
```

Comment:
It looks like you're missing a `*` here to make this line part of the block comment.

Reply:
@matt-thomason Fixed